website/docs/emr-deployments.md

```diff
@@ -0,0 +1,109 @@
+---
+id: emr_deployment
+title: GeoTrellis EMR Deployment with GDAL
+sidebar_label: GeoTrellis EMR Deployment with GDAL
+---
+
+GeoTrellis can be used with _EMR 6_. However, if there is a need in GDAL bindings usage there can be some certain complexities.
+This section describes the entire _EMR 6_ deployment and GDAL installation in a way that would satisfy GeoTrellis GDAL requirements.
+
+Starting _GeoTrellis_ `3.4.2`, _GeoTrellis_ is compatible only with the GDAL`3.1 +` versions.
+
+## Installing GDAL on EMR
+
+To properly install GDAL on EMR it is neccesary to install GDAL on all the nodes using the [EMR bootstrap script](#emr-bootsrap-script-gdal-installation-on-emr). And to make these installed GDAL dependencies
+available for the run-time shared library loader (to set a proper `LD_LIBRARY_PATH` environment variable).
+
+If you're a Scala user and use [SBT](https://www.scala-sbt.org/) for the development work, there a [GeoTrellis Spark Job](https://github.com/geotrellis/geotrellis-spark-job.g8) template project that contains a template for the GeoTrellis project and configures the 
+[SBT Lighter plugin](#configuration-for-sbt-lighter-plugin) to install GDAL.
+
+To summarize, the whole process can be represented the following way:
+
+1. Install _GDAL_
+2. Set Spark `LD_LIBRARY_PATH`. In this case `GeoTrellis GDAL` would be able to use the installed `GDAL`
+
+### EMR Bootsrap script (GDAL installation on EMR)
+
+The EMR bootstrap script for GeoTrellis + GDAL can be downloaded at: [s3://geotrellis-test/emr-gdal/bootstrap.sh](s3://geotrellis-test/emr-gdal/bootstrap.sh).
+
+It installs GDAL 3.1.2 (by default) through conda and sets all appropriate env variables in case 
+it would be required to log in on the cluster nodes and experiment with the `spark-shell`.
+
+#### spark-defaults settings
+
+To make GDAL bindings work it is neccesary to pass the `LD_LIBRARY_PATH` variable into the Spark task.
+To set `LD_LIBRARY_PATH` for spark, through the `spark-defaults` settings it is possible to do the following:
+
+```json
+{
+  "Classification": "spark-defaults",
+  "Properties": {
+    "spark.yarn.appMasterEnv.LD_LIBRARY_PATH": "/usr/local/miniconda/lib/:/usr/local/lib",
+    "spark.executorEnv.LD_LIBRARY_PATH": "/usr/local/miniconda/lib/:/usr/local/lib"
+  }
+}
+```
+
+These parameters above can be also set / overrided via the `spark-submit --conf` settings.
+
+### Configuration for SBT Lighter Plugin
+
+Before looking into this section, you may be interested in the [GeoTrellis Spark Job](https://github.com/geotrellis/geotrellis-spark-job.g8) template project as well.
+
+The SBT Lighter Plugin configuration example:
+
+```scala
```

Comment:
This doesn't seem to give syntax highlight for the block like https://geotrellis.github.io/geotrellis-workshop/docs/layer-model does, guessing this is dependent on the theme?

![Screen Shot 2020-12-16 at 9 06 42 AM](https://user-images.githubusercontent.com/1158084/102358875-09561e00-3f7e-11eb-8f1b-e200821901b7.png)

Reply:
Fixed in https://github.com/locationtech/geotrellis/pull/3317/commits/f0e24a42a5d5afda556e411cde06b431cb7488a9

![Screen Shot 2020-12-16 at 10 33 15 AM](https://user-images.githubusercontent.com/1818302/102369824-622bb380-3f8a-11eb-9475-9340c7ca8b48.png)